src/sagas/certificate.js

```diff
@@ -12,6 +12,7 @@ import { sendEmail } from "../services/email/sendEmail";
 import { processQrCode } from "../services/qrProcessor";
 import { verifyDocument } from "../services/verify";
 import { isValid } from "../services/verify/fragments";
+import { decryptString } from "@govtechsg/oa-encryption";
```

Comment:
where are you using this ?

Reply:
it was already used in the code line 103, `npm run storybook` was complaining it was not defined.